pkg/machinery/config/types/runtime/event_sink.go

```diff
@@ -93,6 +93,11 @@ func (s *EventSinkV1Alpha1) KmsgLogURLs() []*url.URL {
 	return nil
 }
 
+// WatchdogTimer implements config.RuntimeConfig interface.
+func (s *EventSinkV1Alpha1) WatchdogTimer() config.WatchdogTimerConfig {
```

Comment:
hmm why does eventsink need watchdogtimer method implemented?

Reply:
that's the way I implemented `Runtime()` config interface - all documents implement all methods, but return `nil` if it's not their thing. it might be good or bad idea, idk, but it is this way right now ;)